certbot/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * `zope` based interfaces in `certbot.interfaces` module are deprecated and will
   be removed in a future release of Certbot. Any import of these interfaces will
   emit a warning to prepare the transition for developers.
+* We removed the dependency on `chardet` from our acme library.
```

Comment:
Maybe we could be more explicit about the user impact behind this change, by saying that `acme` will no try to guess the encoding anymore, since the encoding is a requirement from the ACME specs.

Reply:
I made that change as well as being more explicit about the fact that setting `response.encoding` affects calling functions as well. I was worried we might some day remove the `response.text` usage in that function and think that setting `response.encoding` was no longer needed.